### PR TITLE
feat: [SRM-14453]: Cypress for Request based SLO

### PR DESCRIPTION
**Pull request by deepesh-ui**
#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

File: cypress/integration/85-cv/slos/RequestBasedSLO.spec.ts [hunk 1]
```diff
@@ -0,0 +1,383 @@
...
+
+    // selecting health source
+    cy.get('input[name="healthSourceRef"]').click()
+    cy.contains('p', 'appd_cvng_prod').click({ force: true })
+
+    // Select Request
+    cy.contains('div', 'REQUEST').click({ force: true })
+
+    // selecting event type
+    cy.get('input[name="eventType"]').click()
```
Comment (praneshg239) [inline cypress/integration/85-cv/slos/RequestBasedSLO.spec.ts:136]: NIT: For inputs shall we stick with `findByRole` and `name`?

```ts
cy.findByRole("textbox", {name: /eventType/i})
```

File: cypress/integration/85-cv/slos/RequestBasedSLO.spec.ts [hunk 2]
```diff
@@ -0,0 +1,383 @@
...
+
+    // Filling details Under Name tab for SLO creation
+    cy.fillName('SLO-1')
+
+    // selecting monitored service
+    cy.get('input[name="monitoredServiceRef"]').click()
+    cy.contains('p', 'cvng_prod').click({ force: true })
+
+    // selecting user journey
+    cy.get('div[data-testid="multiSelectService"]').click()
```
Comment (praneshg239) [inline cypress/integration/85-cv/slos/RequestBasedSLO.spec.ts:361]: for test id use `cy.findByTestid`

Comment (deepesh-ui) [reply cypress/integration/85-cv/slos/RequestBasedSLO.spec.ts:361]: done

File: cypress/integration/85-cv/slos/RequestBasedSLO.spec.ts (hunk 1 above)
Comment (deepesh-ui) [reply cypress/integration/85-cv/slos/RequestBasedSLO.spec.ts:136]: tried didn't work ,Name property coming as empty 

Screenshot 2023-04-04 at 12 23 59 PM

**Review by praneshg239 — APPROVED** [review]

**Review by karansaraswat19 — APPROVED** [review]

**Pull request merged**